### PR TITLE
Refactor gar-t5 experiments doc

### PR DESCRIPTION
**Pull request by manveertamber**
- runs were previously incorrectly named with gar-bart instead of gar-t5
- replaced $ANSERINI/src/main/python/fusion.py with python -m pyserini.fusion
- Added hybrid retrieval results with DKRR to tables

### CODE REVIEW

File: docs/experiments-gar-t5.md [hunk 1]
```diff
@@ -21,47 +27,81 @@ Without specifying the output path, the default output will be an `augmented_top
...
 
 To evaluate the run:
 ```
-python -m pyserini.eval.evaluate_dpr_retrieval --retrieval runs/gar-bart-run.json --topk 1 5 10 20 50 100 200 300 500 1000
+python -m pyserini.eval.evaluate_dpr_retrieval \
+  --retrieval runs/gar-t5-run.json \
+  --topk 1 5 10 20 50 100 200 300 500 1000
+```
+
+To run hybrid sparse and dense retrieval with gar-t5 and [DKRR](https://github.com/castorini/pyserini/blob/master/docs/experiments-dkrr.md):
```
Comment (ronakice) [inline docs/experiments-gar-t5.md:58]: Make this a separate section and point to it at the end of the DKRR docs too?

File: docs/experiments-gar-t5.md [hunk 2]
```diff
@@ -21,47 +27,81 @@ Without specifying the output path, the default output will be an `augmented_top
...
+|    NQ    |      sentence      | 42.00 | 57.78 | 64.12 | 69.59 | 75.62 |  79.67 |  83.03 |  85.04 |  86.87 |  89.00  |
+|    NQ    |        title       | 32.15 | 50.66 | 58.68 | 65.76 | 73.30 |  78.25 |  82.19 |  84.15 |  85.91 |  88.01  |
+|    NQ    |       fusion       | 45.44 | 64.89 | 71.82 | 77.16 | 82.55 |  85.34 |  88.00 |  89.15 |  90.13 |  91.74  |
+|    NQ    | hybrid (with DKRR) | 53.36 | 73.66 | 79.92 | 84.46 | 88.24 |  90.22 |  91.42 |  92.10 |  92.65 |  93.26  |
+| TriviaQA |       answer       | 55.92 | 70.39 | 74.77 | 78.39 | 82.36 |  84.55 |  86.23 |  87.42 |  88.36 |  89.34  |
+| TriviaQA |      sentence      | 49.17 | 63.30 | 68.42 | 72.57 | 77.55 |  80.67 |  83.33 |  84.93 |  86.22 |  87.78  |
+| TriviaQA |        title       | 47.58 | 61.31 | 66.59 | 71.57 | 76.79 |  80.15 |  82.95 |  84.18 |  85.65 |  87.30  |
+| TriviaQA |       fusion       | 59.48 | 73.43 | 77.29 | 80.43 | 83.80 |  85.60 |  87.11 |  87.81 |  88.70 |  89.68  |
+| TriviaQA | hybrid (with DKRR) | 65.81 | 79.40 | 82.34 | 84.69 | 86.87 |  88.05 |  88.99 |  89.52 |  90.05 |  90.61  |
```
Comment (ronakice) [inline docs/experiments-gar-t5.md:90]: Do these numbers match?

File: docs/experiments-gar-t5.md (hunk 1 above)
Comment (ronakice) [reply docs/experiments-gar-t5.md:58]: "hybrid sparse-dense retrieval"

**Review by ronakice — CHANGES_REQUESTED** [review]
Some comments

File: docs/experiments-gar-t5.md (hunk 2 above)
Comment (manveertamber) [reply docs/experiments-gar-t5.md:90]: most match, some are off by 0.01, the largest discrepancy is 0.03 for the top-5 nq-test results for the hybrid

**Review by ronakice — APPROVED** [review]
LGTM!

**Pull request merged**